config/example-config.exs

```diff
@@ -0,0 +1,12 @@
+use Mix.Config
+
+config :hermetic, Hermetic.Router,
+  slack_secrets: ["00000000000000000000000000000000"]
+
+config :hermetic, Hermetic.Slack,
+  token: "xoxa-0000000000-000000000000-000000000000-00000000000000000000000000000000"
+
+# YouTrack's base URL and user token
+config :hermetic, Hermetic.YouTrack,
+  base_url: "https://youtrack.example.com",
+  token: "perm:00000000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
I liked the old `token` example more:

```
iex(1)> Base.decode64("d2h5IGRpZCB5b3UgZGVjb2RlIHRoaXM/IHBsZWFzZSBkbyBub3QgZGVjb2RlIGluIHRo")
{:ok, "why did you decode this? please do not decode in th"}
```

Reply:
I had no idea :p

Reply:
Well, you're probably right, it's much easier to identify `0000...` as example than a random string.